awx/main/tasks.py

```diff
@@ -862,10 +862,12 @@ def build_params_process_isolation(self, instance, private_data_dir, cwd):
                     settings.PROJECTS_ROOT,
                     settings.JOBOUTPUT_ROOT,
                 ] + getattr(settings, 'AWX_PROOT_HIDE_PATHS', None) or [],
-                'process_isolation_ro_paths': [settings.ANSIBLE_VENV_PATH, settings.AWX_VENV_PATH],
             }
             if getattr(instance, 'ansible_virtualenv_path', settings.ANSIBLE_VENV_PATH) != settings.ANSIBLE_VENV_PATH:
-                process_isolation_params['process_isolation_ro_paths'].append(instance.ansible_virtualenv_path)
+                process_isolation_params['process_isolation_ro_paths'] = [instance.ansible_virtualenv_path]
```

Comment:
...aaaaaactually now that I think of this, this might break inventory updates, which probably rely on awx + <custom venv> in certain circumstances cc @AlanCoding

Reply:
Correct, we need that `settings.AWX_VENV_PATH`. I agree with your spirit here - that playbook runs _don't_ need this. Maybe you could conditionally lose that if it's not an inventory update or system job?